app/views/components/docs/published-dates.yml

```diff
@@ -0,0 +1,42 @@
+name: Published dates
+description: Dates to reflect when content was published and updated
+accessibility_criteria: |
+  The published dates component must:
+
+    - indicate to users that the full history section can be expanded and collapsed
+    - inform the user of the state of the full history section (expanded or collapsed)
+    - be usable with a keyboard
+shared_accessibility_criteria:
+  - link
+examples:
+  default:
+    data:
+      published: 1st January 1990
+  just_last_updated_date:
+    data:
+      last_updated: 20th October 2016
+  with_last_updated_date:
+    data:
+      published: 1st January 1990
+      last_updated: 20th October 2016
+  link_to_page_history:
+    description: This will set up a link to a '#history' anchor on the page - this can either be your own defined anchor or another instance of this component, see [Display Page History example](/component-guide/published-dates/display_page_history)
```

Comment:
Doesn't look like the anchor can be altered? I think that's ok though – just this description that needs updating.